Agents/DerivationAgentPythonExample/README.md

```diff
@@ -0,0 +1,231 @@
+# Derivation Agent Python Example
+
+## Purpose
+This agent folder is intended as a template that one can copy and adapt to turn their own Python code into a derivation agent. The example requires [`pyderivationagent`](https://pypi.org/project/pyderivationagent/)>=1.4.1.
+
+This document covers four stages: development, test, package & publish, and deployment. For each stage, a step-by-step instruction is provided. Before continuing with this tutorial, it is recommended to read the [documentation](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_BASE_LIB/python_derivation_agent) of `pyderivationagent` and all the relevant links.
+
+If you identified anything that can be improved to make it easier for newcomers, please feel free to open a [pull request](https://github.com/cambridge-cares/TheWorldAvatar/pulls) or get in touch with the maintainer of the package.
+
+
+&nbsp;
+## Environment setup
+
+For development and testing reasons, follow below instructions to get started.
+
+### Virtual environment setup
+
+It is highly recommended to install `pyderivationagent` packages in a [virtual environment](https://docs.python.org/3/tutorial/venv.html). The following steps can be taken to build a virtual environment:
+
+`(Windows)`
+
+```cmd
+$ python -m venv <venv_name>
+$ <venv_name>\Scripts\activate.bat
+(<venv_name>) $
+```
+
+`(Linux)`
+```sh
+$ python3 -m venv <venv_name>
+$ source <venv_name>/bin/activate
+(<venv_name>) $
+```
+
+The above commands will create and activate the virtual environment `<venv_name>` in the current directory.
+
+### Package installation
+
+The following command can be used to install all required packages.
+
+`(Linux)`
+```bash
+(<venv_name>) $ python -m pip install --upgrade pip
+# Install all required packages from setup.py, incl. pyderivationagent, pytest etc.
+(<venv_name>) $ python -m pip install -r requirements.txt
```

Comment:
Somehow true as `requirements.txt` contains the pinned packages from `setup.py`, but still misleading. I suggest clarifying or aligning

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/391/commits/c872c3b2a6e16f2080a545d71e9ce41d07873366